### PR TITLE
Scg app level routing

### PR DESCRIPTION
**Pull request by leonard520**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
spring

**Comment by yuwzho** [comment]
Update version and changelog in #5532

File: src/spring/setup.py [hunk 1]
```diff
@@ -54,4 +54,4 @@
     packages=find_packages(),
     install_requires=DEPENDENCIES,
     package_data={'azext_spring': ['azext_metadata.json']},
-)
+)
```
Comment (zhoxing-ms) [inline src/spring/setup.py:57]: Why do you modify here?

**Comment by zhoxing-ms** [comment]
Please write the description of PR change into the `HISTORY.rst`

File: src/spring/setup.py (hunk 1 above)
Comment (leonard520) [reply src/spring/setup.py]: reverted

**Comment by leonard520** [comment]
> Please write the description of PR change into the `HISTORY.rst`
It will be covered in https://github.com/Azure/azure-cli-extensions/pull/5532

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**